protocol/x/bridge/module_test.go

```diff
@@ -118,7 +98,7 @@ func TestAppModuleBasic_RegisterInterfaces(t *testing.T) {
 	mockRegistry.On("RegisterImplementations", (*sdk.Msg)(nil), mock.Anything).Return()
 	mockRegistry.On("RegisterImplementations", (*tx.MsgResponse)(nil), mock.Anything).Return()
 	am.RegisterInterfaces(mockRegistry)
-	mockRegistry.AssertNumberOfCalls(t, "RegisterImplementations", 15)
```

Comment:
Can we move away from the `mockRegistry` and instead using `EnsureRegistered()` or one of the other equivalent methods to make sure that all the types we want to have registered are registered here and elsewhere?

This would allow us to remove the `InterfaceRegistry` from `mock-gen` as well.

In Cosmos 0.50.0 you can't mock the registry anymore since it has a private field that is accessed outside of the type.

Feel free to defer doing this.

Reply:
will capture in this issue: https://linear.app/dydx/issue/CORE-557/remove-mockregistry-from-module-testgo